### PR TITLE
builds(auto-pr): Add `CommitMessageFormat` option

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
#### Description

Add `CommitMessageFormat` option to support custom commit message for GitHub action.

Such as: `CommitMessageFormat` : `chore: Upgrade <app> to version <version>`

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

File: bin/auto-pr.ps1 [hunk 1]
```diff
@@ -11,6 +11,9 @@
 .PARAMETER App
     Manifest name to search.
     Placeholders are supported.
+.PARAMETER CommitMessageFormat
+    The format of the commit message.
+    <app> will repalce with the name of the name of manifest.
```
Comment (lewis-yeung) [inline bin/auto-pr.ps1:16]:
```suggestion
    <app> will be replaced with the name of manifest.
```

Comment (yi-Xu-0100) [inline bin/auto-pr.ps1]:
```suggestion
    <app> will replace with the file name of manifest.
    <version> will replace with the version in the latest manifest.
```

**Review by lewis-yeung — DISMISSED** [review]

**Comment by niheaven** [comment]
Change title and move changelog item to `builds`.

**Comment by yi-Xu-0100** [comment]
@niheaven The changelog has been updated. 😁

**Review by lewis-yeung — APPROVED** [review]

**Comment by yi-Xu-0100** [comment]
@lewis-yeung @niheaven 

I found a bug that `$version` was prepared later and will lose the version information in `commitMessage`. and I push a new commit([612169e](https://github.com/ScoopInstaller/Scoop/pull/5171/commits/612169eb78fa107cdfec9984f357614205d7b935)) to fix it. 😅

**Review by lewis-yeung — APPROVED** [review]

**Review by niheaven — APPROVED** [review]

**Pull request merged**